util/microservice-sdk/js/lib/configManager/Manager.js

```diff
@@ -0,0 +1,37 @@
+const Parsers = require('./parsers');
+const { Reader, Writer } = require('./fileManager');
+const Merger = require('./Merger');
+
+/**
+ * Creates the configuration file ./config/<service>.conf. The precedence is (from higher to lower):
+ * - Environment variables
+ * - User config file
+ * - Default config file
+ *
+ * @param {string} service acronym for the service, should only contain letters/numbers
+ * @param {string} path path to the user configuration file, defaults to './config'
+ */
+const createConfig = (service, path = './config', userConfigFile = 'production.conf') => {
+  const envVarsData = Parsers.EnvVars.parseEnvironmentVariables(service);
+  const userData = Reader.readUserConfig(path, userConfigFile);
+  const defaultData = Reader.readDefaultConfig();
+
+  const config = Merger.mergeConfigs(envVarsData, userData, defaultData);
+
+  Writer.writeJson(service, path, config);
+};
```

Comment:
Is it really necessary to save the settings to disk?
If the Kubernetes Pod dies, will the file not be lost?
I think keeping these settings in memory for the life of the application is a more practical approach.

Thinking about it, the name of this function could change to `loadSettings()`